src/components/Promotion/index.js

```diff
@@ -0,0 +1,202 @@
+import React, { useEffect, useState } from 'react';
+import { useRouter } from 'next/router';
+import PromotionContainer from './PromotionContainer';
+import moment from 'moment';
+
+import { getData, getBoardData, getSearchData } from 'apis/promotion';
+
+const PromotionPage = () => {
+  const [openModal, setOpenModal] = useState(false);
+  const [boardData, setBoardData] = useState([]);
+  const [searchItem, setSearchItem] = useState('');
+  const [searchKeyword, setSearchKeyword] = useState('');
+  const [type, setType] = useState('title');
+  const [isSearch, setIssearch] = useState(false);
+  const [search, setSearch] = useState(false);
+  const [letter, setLetter] = useState();
+  const [isLoading, setIsLoading] = useState(false);
+  const router = useRouter();
+
+  let scrollLoading = false;
+  let itemNo = 0;
+
+  function displayedAt(createdAt) {
+    const time = moment(createdAt);
+    const milliSeconds = moment() - time;
```

Comment:
```suggestion
    const milliSeconds = moment() - moment(createdAt);
```
더이상 함수내에서 time 변수는 사용하지 않는 것 같은데 이건 어떤가여?